rpc/client.go

```diff
@@ -65,11 +65,11 @@ func uploadTests(ctx context.Context, client internal.CedarPerformanceMetricsCli
 		if len(test.Artifacts) > 0 {
 			artifacts := make([]*internal.ArtifactInfo, 0, len(test.Artifacts))
 			for _, a := range test.Artifacts {
-				if a.LocalFile != "" {
-					if err = a.Validate(); err != nil {
-						return errors.Wrap(err, "problem validating artifact")
-					}
+				if err = a.Validate(); err != nil {
+					return errors.Wrap(err, "problem validating artifact")
+				}
```

Comment:
Don't we want to validate at the end so that if the path isn't set (which might/should be an error) we can set it?